### PR TITLE
Add support for RN 0.65

### LINKED ISSUE #6: React Native 0.65 Support
@rpavlovs maybe you can share the steps to create a patch for 0.65

### PR DESCRIPTION
**Pull request by rpavlovs**
Closes: #6 

@andyrobert3 could you please test it?

### CODE REVIEW

**Comment by andyrobert3** [comment]
@rpavlovs i think we forgot to update the `patch.js` file here

File: patches/0.65.0/VirtualizedList.js [hunk 1]
```diff
@@ -0,0 +1,2194 @@
...
+    if (dOffset < 0) {
+      return;
+    }
+    // If contentLength has not changed
+    if (contentLength === this._sentEndForContentLength) {
+      return;
+    }
+    const distanceFromEnd = contentLength - visibleLength - offset;
+    // If the distance is so farther than the area shown on the screen
+    if (distanceFromEnd  visibleLength * 1.5) {
```
Comment (andyrobert3) [inline patches/0.65.0/VirtualizedList.js:1528]: `distanceFromEnd >= visibleLength * 1.5`

File: patches/0.65.0/VirtualizedList.js [hunk 2]
```diff
@@ -0,0 +1,2194 @@
...
+      return;
+    }
+    // $FlowFixMe
+    const minimumDistanceFromEnd = onEndReachedThreshold * visibleLength;
+    if (distanceFromEnd  minimumDistanceFromEnd) {
```
Comment (andyrobert3) [inline patches/0.65.0/VirtualizedList.js:1533]: `distanceFromEnd >= minimumDistanceFromEnd`

**Review by andyrobert3 — CHANGES_REQUESTED** [review]

**Comment by ifsnow** [comment]
@rpavlovs Thanks for your PR!

There were many changes in RN 0.65. So, It's necessary to make sure that this fix does not have any issues.

Please wait for a while to check this.

**Comment by ifsnow** [comment]
@rpavlovs @andyrobert3  A new version with support for RN 0.65.x has been released. Hope it works well.

**Pull request closed without merging**